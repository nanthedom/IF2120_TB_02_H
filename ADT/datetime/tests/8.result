ADT/datetime/tests/8.in ADT/datetime/tests/8.out: TRUE
