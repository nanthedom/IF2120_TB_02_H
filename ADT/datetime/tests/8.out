02/02/2000 02:02:03